### PR TITLE
api-ref: pending methods

### PR DESCRIPTION
**Pull request by sumitsaurabh927**
Add pending methods and added one FAQ

### CODE REVIEW

File: api-reference/execution-details/get-execution-details.mdx [hunk 1]
```diff
@@ -4,6 +4,16 @@ openapi: get /v1/execution-details
...
 
+```javascript Node.js
+import fetch from 'node-fetch';
+
+const response = await fetch('https://api.novu.co/v1/execution-details', {
+  method: 'GET',
+  
+});
+const data = await response.json();
+```
```
Comment (unicodeveloper) [inline api-reference/execution-details/get-execution-details.mdx:15]: This is not the Node SDK method for this. Don't we have that?

File: api-reference/inbound-parse/validate-the-mx-record-setup-for-the-inbound-parse-functionality.mdx [hunk 2]
```diff
@@ -4,6 +4,16 @@ openapi: get /v1/inbound-parse/mx/status
...
+```javascript Node.js
+import fetch from 'node-fetch';
+
+const response = await fetch('https://api.novu.co/v1/inbound-parse/mx/status', {
+  method: 'GET',
+  
+});
+const data = await response.json();
+```
+
```
Comment (unicodeveloper) [inline api-reference/inbound-parse/validate-the-mx-record-setup-for-the-inbound-parse-functionality.mdx:16]: This is not the Node SDK method for this. Don't we have that?

**Review by unicodeveloper — COMMENTED** [review]
Please attend to the comments

File: api-reference/execution-details/get-execution-details.mdx (hunk 1 above)
Comment (sumitsaurabh927) [reply api-reference/execution-details/get-execution-details.mdx:15]: No, the link you sent me didn't have it. And so, as you said, I copied it from the old docs.

File: api-reference/inbound-parse/validate-the-mx-record-setup-for-the-inbound-parse-functionality.mdx (hunk 2 above)
Comment (sumitsaurabh927) [reply api-reference/inbound-parse/validate-the-mx-record-setup-for-the-inbound-parse-functionality.mdx:16]: No, the link you sent me didn't have it. And so, as you said, I copied it from the old docs.

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**